dynamic-plugin/src/components/GenieWidgetsPage.tsx

```diff
@@ -127,8 +132,36 @@ export default function GenieWidgetsPage() {
         />
       </div>
       <div className="dashboard">
+      <div style={{ marginTop: '24px', padding: '24px' }}>
+        {Array.isArray(nguiMock?.blocks) &&
+          nguiMock.blocks.map((block: any) => {
+            const dataType = block?.configuration?.data_type ?? '';
+            const fields = block?.configuration?.component_metadata?.fields ?? [];
+            const config = dataTypeRegistry[dataType];
+            if (config) {
+              return (
+                <div key={block?.id || dataType}>
+                  <ResourceListWidget
+                    config={config}
+                    fields={fields}
+                    // TODO: get the k8s group, version, kind, namespaced, namespace from the NGUI block configuration
+                    k8s={{
```

Comment:
@lkrzyzanek can you provide these details in the NGUI response? We need it to query the API for the live data.